drgn/helpers/linux/idr.py

```diff
@@ -7,21 +7,101 @@
 
 The ``drgn.helpers.linux.idr`` module provides helpers for working with the IDR
 data structure in :linux:`include/linux/idr.h`. An IDR provides a mapping from
-an ID to a pointer. This currently only supports Linux v4.11+; before this,
-IDRs were not based on radix trees.
+an ID to a pointer.
 """
 
 from typing import Iterator, Tuple
 
-from _drgn import _linux_helper_idr_find as idr_find
-from drgn import Object
+from _drgn import _linux_helper_idr_find as idr_find_in_radix_tree
+from drgn import NULL, Object, Program, sizeof
 from drgn.helpers.linux.radixtree import radix_tree_for_each
 
 __all__ = (
     "idr_find",
     "idr_for_each",
 )
 
+IDR_BITS = 8
+IDR_MASK = (1 << IDR_BITS) - 1
```

Comment:
Let's indicate that these are internal:

```suggestion
_IDR_BITS = 8
_IDR_MASK = (1 << _IDR_BITS) - 1
```

Reply:
Done